xpdacq/beamtime.py

```diff
@@ -282,11 +285,23 @@ def __init__(self,name, scanplan_type, scanplan_params, dk_window = None, shutte
         subs=[]
         if livetable:
             subs.append('livetable')
+        else:
```

Comment:
since we have now removed the ``subs`` from ScanPlan, I think we can clean the ``subs`` stub from the autonaming scheme.